scripts/datascience/themes_quality/config/pipeline_config.yaml

```
@@ -0,0 +1,148 @@
+  include_metadata: true
+  include_confidence_scores: true
+  include_alternative_categories: true
+  sort_by_confidence: true
+  
+# Data Paths
+paths:
+  input_vocab: "data/filtered_vocabulary.json"
+  unigram_freq: "data/unigram_freq.csv"
+  output_dir: "data/vocabs"
+  temp_dir: "data/temp"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)